src/index.html

```diff
@@ -1,12 +1,198 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <section class="calendar calendar--month-length--29 calendar__start-day--sun">
+      <div 
+        class="calendar__days calendar__days--1"
+      >
+      </div>
+
+
+      <div 
+        class="calendar__days calendar__days--2"
+      >
+      </div>
+
+
+      <div 
+        class="calendar__days calendar__days--3"
+      >
+      </div>
+
+
+      <div 
+        class="calendar__days calendar__days--4"
```

Comment:
```suggestion
     <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

нумерацію задаємо через стилі
<img width="593" alt="image" src="https://user-images.githubusercontent.com/102413088/187231078-d0d3c215-f266-4c25-81ab-7ec9cd030ad3.png">

Reply:
так она и создана через стиль 

@for $index from 1 through 31 {
      &__days--#{$index}::before{
        content: "#{$index}";
      }
    }.  (calendar.scss 11-15 строка)